utt/libutt/test/TestIBE.cpp

```diff
@@ -65,8 +66,10 @@ int main(int argc, char *argv[]) {
   auto ctxtCopy = ctxt;
   testAssertEqual(ctxt, ctxtCopy);
 
-  testAssertNotEqual(ctxt,
-                     mpk.encrypt(pid, frsToBytes({Fr::random_element(), Fr::random_element(), Fr::random_element()})));
+  auto frs_data = frsToBytes({Fr::random_element(), Fr::random_element(), Fr::random_element()});
+  AutoBuf<unsigned char> auto_buf_data(frs_data.size());
+  memcpy(auto_buf_data.getBuf(), encbuf.data(), encbuf.size());
```

Comment:
This line seems wrong and is failing the IBE test.

Reply:
fixed